var/spack/repos/builtin/packages/picsar/package.py

```diff
@@ -65,6 +69,9 @@ def build_targets(self):
 
         targets.append('SYS = default')
 
+        if '%gcc' in self.spec:
+            targets.append('FARGS=-g -fbounds-check -O3 -fopenmp -JModules -fallow-argument-mismatch')
```

Comment:
This line is causing the flake8 tests to fail:
```
  var/spack/repos/builtin/packages/picsar/package.py:73: [E501] line too long (102 > 88 characters)
```
Can you break it into multiple lines?